studio/schemas/documents/compensations.ts

```diff
@@ -15,74 +12,59 @@ const compensations = defineType({
   type: "document",
   title: "Compensations",
   fields: [
-    locations,
-    title,
+    {
+      ...title,
+      title: "Compensation Page Title",
+      description:
+        "Enter the primary title that will be displayed at the top of the compensation page. This is what users will see when they visit the page.",
+    },
     titleSlug,
-    seo,
-    defineField({
-      name: "showSalaryCalculator",
-      title: "Show Salary Calculator",
-      description: "Should the salary calculator be visible on the page?",
-      type: "boolean",
-      initialValue: true,
-    }),
+    // add pension here. pension doesn't rely on locations
+    bonusesByLocation,
+    // add salary here
```

Comment:
I want to keep these comments so we know what's expected here once we add pension, benefits and salary